### PR TITLE
Github: Removed chained promises in server.js and simplified code in service.js

### PR DESCRIPTION
**Pull request by manekenpix**
Github: Removed chained promises in server.js and simplified code in service.js

### CODE REVIEW

File: Github/service.js [hunk 1]
```diff
@@ -13,206 +13,124 @@ var request = require("request");
...
+                    rep.branches.forEach(branch =>{
+                        promises.push(getCommits({repo: rep.repo, br: branch}));
+                    });
+                });
+                Promise.all(promises)
+                .then(()=>{
+                    recentCommits.sort((a, b) => {
+                        let c = new Date(a.commit.author.date);
+                        let d = new Date(b.commit.author.date);
+                        return d - c;
```
Comment (belavina) [inline Github/service.js]: `a` & `c` could be `const`? Also maybe update variable names `a`, `b`, `c`, `d` for clarity

File: Infrastructure/workstations.json [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
-    {"Host":"switzerland", "IPAddress":"67.69.105.132", "Status":"unknown"},
-    {"Host":"japan", "IPAddress":"67.69.105.139", "Status":"unknown"}
+    {"Host":"bbb", "IPAddress":"bbb.cdot.systems", "Status":"unknown"},
+    {"Host":"vega", "IPAddress":"vega.cdot.systems", "Status":"unknown"},
+    {"Host":"rob", "IPAddress":"rob.cdot.systems", "Status":"unknown"}, 
+    {"Host":"ps", "IPAddress":"ps.cdot.systems", "Status":"unknown"},
+    {"Host":"italy", "IPAddress":"italy.cdot.systems", "Status":"unknown"},
+    {"Host":"canada", "IPAddress":"canada.cdot.systems", "Status":"unknown"},
+    {"Host":"narnia", "IPAddress":"narnia.cdot.systems", "Status":"unknown"},
+    {"Host":"ireland", "IPAddress":"ireland.cdot.systems", "Status":"unknown"},
```
Comment (belavina) [inline Infrastructure/workstations.json]: Can you remove `ireland` (it's Yan's machine)?

File: Github/service.js (hunk 1 above)
Comment (manekenpix) [reply Github/service.js]: Sorry, I was just reusing code

File: Infrastructure/workstations.json (hunk 2 above)
Comment (manekenpix) [reply Infrastructure/workstations.json]: Removed

File: Github/service.js (hunk 1 above)
Comment (manekenpix) [reply Github/service.js]: variable names changed

**Pull request merged**